src/browser.ts

```diff
@@ -74,17 +76,21 @@ export class GrpcClient {
       headers['Content-Type'] = 'application/x-protobuf';
       headers['User-Agent'] = 'testapp/1.0';
 
+      //const grpc_fallback_protocol = opts.protocol | 'https'
       const servicePath =
         opts.servicePath || method.parent.options['(google.api.default_host)'];
       const servicePort = opts.port || 443;
-      const serviceName = method.parent.parent.options.java_package.substring(
-        4
+      const serviceName = method.parent.parent.options.java_package.replace(
+        /^com./, ''
       );
       const rpcNamespace = method.parent.name;
       const rpcName = method.name;
 
-      const url = `https://${servicePath}:${servicePort}/$rpc/${serviceName}.${rpcNamespace}/${rpcName}`;
+      // const url = `https://${servicePath}:${servicePort}/$rpc/${serviceName}.${rpcNamespace}/${rpcName}`;
+
+      const url = `http://${servicePath}:${servicePort}/$rpc/${serviceName}.${rpcNamespace}/${rpcName}`;
```

Comment:
Do we want this to be 'http', I know we need it for the gapic-showcase for now but the Google APIs will use https. Is your commented code on line 79 `//const grpc_fallback_protocol = opts.protocol | 'https'` have anything to do with this?

Reply:
Good catch! It must be `https` by default and it must be possible to override it using an option (to make it work with showcase).

Reply:
Yeah, that comment is related to this issue... when I looked into fixing this stuff last week, it seemed like there had to be changes in multiple places (although I could be mistaken) so I kind of left it off for later. Do you think it would be better to do it properly right now, as part of this pull request? If so, I'll try and make that change

Reply:
Yes, since the `http` is only needed for showcase tests, this change actually belongs here in this pull request. It's a small change and it should not conflict with @LibanOdowa's changes (even if there is a conflict it will be easy to resolve).